orm/models.py

```diff
@@ -241,6 +241,12 @@ def limit(self, limit_count: int):
         instance.pk = await self.database.execute(expr)
         return instance
 
+    async def get_or_create(self, **kwargs):
+        try:
+            return await self.get(**kwargs)
```

Comment:
Should we be using `.filter(**kwargs).exists()` here?

(And should `exists` be adding a `limit` clause?)

Reply:
Using `exists` here would result in an additional round trip to the database.

I don't think and exists query requires a limit. I think that's implicit.